indigo/indigo/src/main/scala/indigo/shared/scenegraph/Graphic.scala

```diff
@@ -51,53 +53,9 @@ final case class Graphic(
   def modifyMaterial(alter: Material => Material): Graphic =
     this.copy(material = alter(material))
 
-  def moveTo(pt: Point): Graphic =
-    this.copy(position = pt)
-  def moveTo(x: Int, y: Int): Graphic =
-    moveTo(Point(x, y))
-  def withPosition(newPosition: Point): Graphic =
-    moveTo(newPosition)
-
-  def moveBy(pt: Point): Graphic =
-    this.copy(position = position + pt)
-  def moveBy(x: Int, y: Int): Graphic =
-    moveBy(Point(x, y))
-
-  def rotateTo(angle: Radians): Graphic =
-    this.copy(rotation = angle)
-  def rotateBy(angle: Radians): Graphic =
-    rotateTo(rotation + angle)
-  def withRotation(newRotation: Radians): Graphic =
-    rotateTo(newRotation)
-
-  def scaleBy(amount: Vector2): Graphic =
-    this.copy(scale = scale * amount)
-  def scaleBy(x: Double, y: Double): Graphic =
-    scaleBy(Vector2(x, y))
-  def withScale(newScale: Vector2): Graphic =
-    this.copy(scale = newScale)
-
-  def transformTo(newPosition: Point, newRotation: Radians, newScale: Vector2): Graphic =
-    this.copy(position = newPosition, rotation = newRotation, scale = newScale)
-
-  def transformBy(positionDiff: Point, rotationDiff: Radians, scaleDiff: Vector2): Graphic =
-    transformTo(position + positionDiff, rotation + rotationDiff, scale * scaleDiff)
-
   def withDepth(newDepth: Depth): Graphic =
     this.copy(depth = newDepth)
 
-  def flipHorizontal(isFlipped: Boolean): Graphic =
-    this.copy(flip = flip.withHorizontalFlip(isFlipped))
-  def flipVertical(isFlipped: Boolean): Graphic =
-    this.copy(flip = flip.withVerticalFlip(isFlipped))
-  def withFlip(newFlip: Flip): Graphic =
-    this.copy(flip = newFlip)
-
-  def withRef(newRef: Point): Graphic =
-    this.copy(ref = newRef)
-  def withRef(x: Int, y: Int): Graphic =
-    withRef(Point(x, y))
-
```

Comment:
This all looks great, but I'm surprised there's no reference to the Sandbox or Perf projects in the PR. Are they just working magically because they're implemented as extension methods?

Reply:
It is building without errors so far :thinking:  running `build.sh` from the root folder (repo root to be more specific, not the indigo folder).